storage/storage_helpers.cpp

```diff
@@ -18,4 +17,22 @@ bool IsDownloadFailed(Status status)
   return status == Status::EDownloadFailed || status == Status::EOutOfMemFailed ||
          status == Status::EUnknown;
 }
+
+m2::RectD CalcLimitRect(TCountryId countryId,
+                        Storage const & storage,
+                        CountryInfoGetter const & countryInfoGetter)
+{
+  m2::RectD boundingBox;
+  auto const accumulater =
```

Comment:
There is no such word as "accumulater". Use "accumulator", or, more specific, "addRect".